diff --git a/config/qman.conf b/config/qman.conf
index 729fd80..6a4e66d 100644
--- a/config/qman.conf
+++ b/config/qman.conf
@@ -15,7 +15,7 @@ enable=             true
 
 ; Users of `mandb` version 2.10 or earlier (use `man --version`) should
 ; uncomment the following line
-; legacy_mandb=       true
+legacy_mandb=       true
 
 ; If using a graphical web browser, uncomment this to suppress an annoying
 ; flicker after opening an HTTP link
